### PR TITLE
[12.0][ADD] fix invoice tax rounding

### PR DESCRIPTION
**Pull request by sergiocorato**

### CODE REVIEW

**Comment by emagdalenaC2i** [comment]
LGTM
Without this addin I get this:
image
And with this addin this:
image

File: account_invoice_fix_tax_rounding/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Sergio Corato <https://github.com/sergiocorato>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Fix invoice tax rounding',
+    'version': '12.0.1.0.0',
+    'category': 'Accounting & Finance',
+    'license': 'AGPL-3',
+    'summary': 'Fix invoice tax rounding',
+    'website': 'https://efatto.it',
```
Comment (emagdalenaC2i) [inline account_invoice_fix_tax_rounding/__manifest__.py]:
```suggestion
    'website': 'https://github.com/OCA/account-invoicing',
```

**Review by emagdalenaC2i — CHANGES_REQUESTED** [review]

File: account_invoice_fix_tax_rounding/__manifest__.py (hunk 1 above)
Comment (sergiocorato) [reply account_invoice_fix_tax_rounding/__manifest__.py]: :+1:

**Review by emagdalenaC2i — APPROVED** [review]

**Comment by emagdalenaC2i** [comment]
@OCA/accounting-maintainers  please, could add the needs_review label to this PR?

**Review by GSLabIt — APPROVED** [review]
LGTM

**Comment by emagdalenaC2i** [comment]
@OCA/accounting-maintainers please, could you review this PR?

**Comment by emagdalenaC2i** [comment]
@OCA/accounting-maintainers please, could you review this PR?

**Comment by dreispt** [comment]
Hello, I understand there is some background discussion on this.
Can someone point me to it?
Also, the readme description is not very helpful to understand what this fix does :-(

Thanks!

**Comment by emagdalenaC2i** [comment]
@dreispt Please, check the example I post https://github.com/OCA/account-invoicing/pull/704#issuecomment-625755729

There is an issue in the tax calculation without this module

**Pull request merged**